web3_proxy/src/rpcs/request.rs

```diff
@@ -27,7 +27,7 @@ pub enum OpenRequestResult {
 
 /// Make RPC requests through this handle and drop it when you are done.
 /// Opening this handle checks rate limits. Developers, try to keep opening a handle and using it as close together as possible
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
This cannot be changed to clone. This will not clone properly and will cause problems with rate limiting and active request counting. Why did you add this?

Reply:
I was using the same request handle to make multiple requests (within payment). I changed it now to spawn multiple handles (one per request)

Reply:
We should still remove Clone on this though.

Reply:
ah yes fixed the clone dependency but forgot to remove it from the derive, I removed this now